test/plugin/base_test.rb

```diff
@@ -1360,30 +1395,54 @@ def test_http_request_skip_setting_latency
   # Verify the default and customization of LogEntry field extraction key.
 
   def test_log_entry_insert_id_field
-    verify_field_key('insertId', DEFAULT_INSERT_ID_KEY, 'custom_insert_id_key',
-                     CONFIG_CUSTOM_INSERT_ID_KEY_SPECIFIED, INSERT_ID)
+    verify_field_key('insertId',
+                     default_key: DEFAULT_INSERT_ID_KEY,
+                     custom_key: 'custom_insert_id_key',
+                     custom_key_config: CONFIG_CUSTOM_INSERT_ID_KEY_SPECIFIED,
+                     sample_value: INSERT_ID)
+  end
+
+  def test_log_entry_labels_field
+    verify_field_key('labels',
+                     default_key: DEFAULT_LABELS_KEY,
+                     custom_key: 'custom_labels_key',
+                     custom_key_config: CONFIG_CUSTOM_LABELS_KEY_SPECIFIED,
+                     sample_value: COMPUTE_PARAMS[:labels].merge(
```

Comment:
[Optional] I'd put the whole `COMPUTE_PARAMS[:labels].merge(LABELS_MESSAGE)` on the next line, with a backslash.

Reply:
I would prefer not using backslash unless it's absolutely necessary.